ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/DefaultSynchronizer.java

```diff
@@ -164,6 +166,8 @@ private void handleFastSyncResult(final FastSyncState result, final Throwable er
 
   private void startFullSync() {
     fullSyncDownloader.start();
+    // TODO: enable when pruning cli implemented
+    // pruner.start();
```

Comment:
I would omit this comment for now.  Or possibly just use a hard-coded feature toggle (aka a `private static boolean PRUNING_ENABLED = false`) which we'd later replace with a CLI feature toggle.